src/app/_helpers/versionHelpers.ts

```diff
@@ -11,14 +11,8 @@ export const sdkVersion = "8.7";
 
 export function compareSdkPCoreVersions() {
 
-  const theConstellationVersion = PCore.getPCoreVersion();
-
-  if (theConstellationVersion.includes(sdkVersion)) {
-     
-    console.log(`sdkVersion: ${sdkVersion} matches PCore version: ${PCore.getPCoreVersion()}`);
-  } else {
-     
-    console.error(`sdkVersion: ${sdkVersion} does NOT match PCore version: ${PCore.getPCoreVersion()}`);
-  }
+  // const theConstellationVersion = PCore.getPCoreVersion();
 
+  // eslint-disable-next-line no-console
+  console.warn(`Using PCore version: ${PCore.getPCoreVersion()}. Confirm this is the same version as your Infinity server.`);
```

Comment:
It may be confusing for the user if they cannot directly control the version of PCore they use, it is better to replace the term PCore with something that they do control, i.e. Constellation, since setting the Constellationjs tag determines the PCore version being used. Please correct me if I am wrong. 

The colon also seems unnecessary here.

I suggest using the following, instead:

"Using Constellation version ${PCore.getPCoreVersion()}. Ensure this is the same version as your Pega Infinity server."